pi.py

```diff
@@ -12,5 +12,5 @@
     # import newer version

```

Comment:
In general it's not a good idea to add comments for the following reasons:
1. Along with code you also have to now maintain comments. If you or someone else changes this code in the future, but forgets to change the comment, then the person after that will be confused reading it.
2. Code has to be readable by itself. If you feel the need to add comments, then there is most likely a better way to write that code. For example instead of adding a comment called "set number of digits", you could create a new function called set_number_of_digits() and put the logic inside it. This eliminates the need to add a comment.
3. You also could add a test that documents the logic. For example, you could write a test that verifies that the number of digits are set correctly. That would indicate to the reader what the code is doing.

Reply:
Hmm makes sense